code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -148,3 +148,32 @@
 	name = "science crate"
 	desc = "A science crate."
 	icon_state = "scicrate"
+
+/obj/structure/closet/crate/solarpanel_small
+	name = "budget solar panel crate"
+	icon_state = "engi_e_crate"
+
+/obj/structure/closet/crate/solarpanel_small/PopulateContents()
+	..()
+	for(var/i in 1 to 13)
+		new /obj/item/solar_assembly(src)
+	new /obj/item/circuitboard/computer/solar_control(src)
+	new /obj/item/paper/guides/jobs/engi/solars(src)
+	new /obj/item/electronics/tracker(src)
+
+/obj/structure/closet/crate/goldcrate
+	name = "gold crate"
+
+/obj/structure/closet/crate/goldcrate/PopulateContents()
+	..()
+	for(var/i in 1 to 3)

```

Comment:
These spawn as a single stack with 3 gold bars instead of three individual bars. No clue how to prevent this.

Reply:
`new /obj/item/stack/sheet/mineral/gold(src, merge=FALSE)`

Or if you want to explicitly stack it rather than running the automerge code:

`new /obj/item/stack/sheet/mineral/gold(src, 3)`

Reply:
forgot `new` doesn't work with keyword arguments, will need to write `(src, 1, FALSE)`

Reply:
Alright, that worked. By the way, is there a quick way to find out where `new`, `PopulateContents` etc. are defined?

Reply:
Best ways are a `grep` variant to search the file tree or something like https://marketplace.visualstudio.com/items?itemName=platymuus.dm-langclient